lib/foreman_maintain/concerns/system_helpers.rb

```diff
@@ -85,6 +85,10 @@ def file_exists?(filename)
         File.exist?(filename)
       end
 
+      def directory_exists(dirname)
```

Comment:
I'd suggest to avoid extending this helper concern unless inevitable. It extends all executable defs and a few other places and adding too much stuff to their scope does not seem like a good practice. This increases risk of conflits and leads to too much methods in the objects. What added value this helper has? Could it be avoided at all?

Reply:
Yes, there is no value by adding this method to helper file instead I can use `File.directory?` method directly. 
I thought similar to `file_exists?` method I should move this method here so moved to helper file.

Updated PR by reverting these changes and using `File.directory?` .